### PR TITLE
What's new: OTel JVMs page (Publish 02-17-22)

### PR DESCRIPTION
**Pull request by rkbodenner**
**Not ready** for publication, as the feature is not yet released in production. Will comment on this PR when ready, which is expected to be Feb. 16 or 17.

No PMM review needed.

### CODE REVIEW

File: src/content/whats-new/2022/02/whats-new-02-15-otel-jvms.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: 'New OTel JVMs page'
```
Comment (daynaslord) [inline src/content/whats-new/2022/02/whats-new-02-15-otel-jvms.md:2]: Switch the title and summary

Change summary to 'A new OTel JVMs page'

File: src/content/whats-new/2022/02/whats-new-02-15-otel-jvms.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+---
+title: 'New OTel JVMs page' 
+summary: 'Compare JVM performance for OTel services'
```
Comment (daynaslord) [inline src/content/whats-new/2022/02/whats-new-02-15-otel-jvms.md:3]: Change title to 'Compare JVM performance metrics for OTel services on a new JVM page'

**Review by daynaslord — DISMISSED** [review]
Just a change to the title and summary. otherwise is looks good to go out whenever @rkbodenner is ready

File: src/content/whats-new/2022/02/whats-new-02-15-otel-jvms.md (hunk 1 above)
Comment (rhetoric101) [reply src/content/whats-new/2022/02/whats-new-02-15-otel-jvms.md:2]: @daynaslord, I switched the title and summary entries like this:

title: 'Compare JVM performance for OpenTelemetry services'
summary: 'New OpenTelemetry JVMs page'

File: src/content/whats-new/2022/02/whats-new-02-15-otel-jvms.md (hunk 2 above)
Comment (rhetoric101) [reply src/content/whats-new/2022/02/whats-new-02-15-otel-jvms.md:3]: @daynaslord, this new change seems to conflict with the previous change where the word "New" is used in the summary. Can we just stick with your previous suggestion?

Comment (daynaslord) [reply src/content/whats-new/2022/02/whats-new-02-15-otel-jvms.md:3]: Sure, we can just change the title to 'Compare JVM performance metrics for OTel services'

**Pull request merged**